eq-author/src/components/modals/PasteModal/index.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect, useCallback } from "react";
+import styled from "styled-components";
+import { colors } from "constants/theme";
+import Modal from "components/modals/Modal";
+import ButtonGroup from "components/buttons/ButtonGroup";
+import Button from "components/buttons/Button";
+import PropTypes from "prop-types";
+
+const StyledModal = styled(Modal)`
+  .Modal {
+    max-width: 500px;
+    border-radius: 0.4rem;
+  }
+`;
+
+const Title = styled.h2`
+  color: ${colors.darkGrey};
+`;
+
+const Message = styled.div``;
+
+const ButtonGroupStyled = styled(ButtonGroup).attrs({
+  horizontal: true,
+  align: "right",
+})`
+  padding-top: 1.5em;
+`;
+
+export const FormatText = (value) => {
```

Comment:
```
export const FormatText = (value) => {
```
As `FormatText` is a function, this should start with a lower case letter `formatText` for all of its instances.

Also, moving this function to its own file as a utility function (eq-author/utils) would improve its reusability - this would keep the function in the same directory as other functions for future development.

Reply:
Refactored as suggested, and renamed to the more clear 'reduceMultipleSpaces'